scripts/watchBackedUpNode.js

```diff
@@ -0,0 +1,104 @@
+var assert = require("assert");
+var fs = require("fs");
+var { promisify } = require("util");
+var readFile = promisify(fs.readFile);
+var ethers = require("ethers");
+
+const SIGNER_ADDRESS = "0x522df396ae70a058bd69778408630fdb023389b2";
+const PORT1 = "8543";
+const PORT2 = "8546";
+const RETRY_TIMEOUT_SECONDS = 2;
+const SCAN_INTERVAL_SECONDS = 5;
+const MAX_VALIDATOR_SET_SIZE = 21;
+
+var Web3 = require("web3");
+var web3_1 = new Web3(new Web3.providers.HttpProvider("http://localhost:" + PORT1));
+var web3_2 = new Web3(new Web3.providers.HttpProvider("http://localhost:" + PORT2));
+var provider = new ethers.providers.JsonRpcProvider("http://localhost:" + PORT2);
+// `true` if the primary is required to sign and `false` if the secondary does.
+var primaryHasToSign = true;
+
+async function scanBlocks() {
+    var lastBlock = await web3_2.eth.getBlock("latest");
+    var lastBlockNum = lastBlock.number;
+    assert(typeof lastBlockNum === "number");
+    if (lastBlockNum < MAX_VALIDATOR_SET_SIZE) {
```

Comment:
Probably `MAX_VALIDATOR_SET_SIZE` should be dynamically calculated based on current validator set size, otherwise smaller validator sets will have a disadvantage, because they will have some of their past blocks in 21 last blocks, even if they stopped produce any new blocks.